### PR TITLE
Make image created with no arguments mutable and add fill method.

### LINKED ISSUE #387: Usage of Image() without parameters
Currently, Image() without parameters creates an empty immutable image. This seems not to be that useful behavior; displaying such image is equivalent to calling display.clear().
When programming simple games, it's necessary to draw to an off-screen buffer and display the buffer with display.show(buffer) when done to avoid flickering. A logical way to write this is:

buffer = Image()
...do what is necessary using buffer.set_pixel()...
display.show(buffer)

However, this raises an exception. The following code works as intended:

buffer = Image(5, 5)
...do what is necessary using buffer.set_pixel()...
display.show(buffer)

Given that the first snippet looks very natural (and is what I tried first), I wonder whether it wouldn't be better if the default width and height were 5. Took a bit of effort (not significant, though) to explain to kids that they need to write Image(5, 5).

### PR DESCRIPTION
**Pull request by markshannon**
This now works
```python
i = Image()
i.set_pixel(2,2,9)
```
The fill method sets all the pixels to the same value

Fixes #387 and #388

### CODE REVIEW

**Comment by markshannon** [comment]
Done

File: docs/image.rst [hunk 1]
```diff
@@ -100,6 +100,13 @@ Classes
         Return a new image by inverting the brightness of the pixels in the
         source image.
 
+    .. py:method:: fill(value)
+
+        Set the brightness of all the pixels in the image to the
+        ``value``, which has to be between 0 (dark) and 9 (bright).
+
+        This method will raise an exception when called on any of the build-in
```
Comment (dpgeorge) [inline docs/image.rst]: Shouldn't it be "built-in"? (note t)

Comment (markshannon) [inline docs/image.rst]: The perils of copy-and-paste. Fixed.

**Comment by dpgeorge** [comment]
Great!

**Pull request merged**